common/static/coffee/spec/discussion/view/discussion_view_spec_helper.coffee

```diff
@@ -23,6 +23,14 @@ class @DiscussionViewSpecHelper
         }
         $.extend(thread, props)
 
+    @checkVoteClasses = (view) ->
+        view.render()
+        display_button = view.$el.find(".display-vote")
+        expect(display_button.hasClass("is-hidden")).toEqual(true)
+        action_button = view.$el.find(".action-vote")
+        # Check that inline css is not applied to the ".action-vote"
+        expect(action_button).not.toHaveClass("inline")
```

Comment:
Inline css is applied using `style` attribute, is this a right way to check for inline css?